resources/views/layout/header.blade.php

```diff
@@ -0,0 +1,8 @@
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
<script> tags should not be in header, they should be in the footer, otherwise there might be a problem executing a script